core-api/src/main/java/com/optimizely/ab/event/internal/BuildVersionInfo.java

```diff
@@ -30,13 +30,25 @@
 /**
  * Helper class to retrieve the SDK version information.
  */
-@Immutable
 public final class BuildVersionInfo {
 
     private static final Logger logger = LoggerFactory.getLogger(BuildVersionInfo.class);
 
+    @Deprecated
     public final static String VERSION = readVersionNumber();
 
+    // can be overridden by other wrapper client (android-sdk, etc)
+
+    private static String clientVersion = readVersionNumber();
```

Comment:
final?

Reply:
Unlike the old VERSION, this private variable can be changed with Optimizely.Builder.